makefile

```diff
@@ -33,7 +33,9 @@ clear-build:  ## Remove precedent build files
 	@rm -rf ./build/*
 
 build: clear-build ## Build all front applications defined with webpack
-	@./node_modules/.bin/webpack --progress
+	@./node_modules/.bin/webpack \
+		$(if $(filter production staging,$(NODE_ENV)),-p,-d) \
```

Comment:
I'm wondering if you shouldn't use a blacklist approach instead, to exclude only `development`. Because even if it makes tests longer, it may help discover bugs that appear only in the optimized version (i've discovered such bugs in admin-on-rest)